hazelcast/src/test/java/com/hazelcast/internal/util/StringUtilTest.java

```diff
@@ -20,8 +20,8 @@
 import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.annotation.ParallelJVMTest;
 import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
```

Comment:
In this form it won't work:
- Junit 4 test must be public
- We use also ParameterizedTest in this class

Reply:
You are right, I miss it. Do you suggest rewrite the test with Parameterized runner?